.github/workflows/maven.yml

```diff
@@ -15,16 +12,15 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
-      with:
-        java-version: 1.8
-    - name: Cache Maven Dependencies
-      uses: actions/cache@v1
+    - name: Checkout repo
+      uses: actions/checkout@v3
+
+    - name: Configure Java
+      uses: actions/setup-java@v3
       with:
-        path: ~/.m2
-        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
-        restore-keys: ${{ runner.os }}-m2
+        java-version: 8
+        cache: 'maven'
+        distribution: temurin
```

Comment:
I had to lookup the usage of single-quotes for these actions and it looks like it just comes down to basic yaml syntax rules of which (of course) there are many thoughts on what "right" is. I don't have a strong preference but I can say that the github action examples I looked through standardize on single quotes for all strings. Not knowing any of this, I would have probably picked no quotes for simplicity. 

Anyway, I got confused because it was done both ways in the PR and the numeric value made me curious if it would work, so I leave it to you to pick a way and we'll try to stick to that convention. This would apply to java-version, cache, and distribution.

Reply:
Very fair point - I'll amend to be single quotes throughout.